packages/core/src/components/Form/pin.module.css

```diff
@@ -1,46 +1,216 @@
-@import "common.module.css";
+/* * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * */
+/* THIS IS GENERATED CODE. Any changes you make will be overwritten! */
+/* * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * */
 
-.pin-field {
-  display: flex;
-  gap: var(--margin-3);
+.cds-form__label {
+  display: block;
+  margin-bottom: var(--cds-form-input-gap);
+  max-width: fit-content;
+  color: var(--cds-form-label-color);
+  font: var(--cds-form-input-text-label);
 }
-
-.form-input {
-  composes: form-input from "common.module.css";
+.cds-form__label.cds-form--disabled {
+  color: var(--cds-form-disabled-label-color);
 }
 
-.form-field {
-  composes: form-field from "common.module.css";
+.cds-form {
+  /* HINT */
```

Comment:
Is this expected? I'm seeing this in multiple places.

Reply:
The `composes:` bits not being there are expected for the time being. This is a bit that we’ll/I’ll need to recreate in the “comprehensive” stylesheet someone would use if they were using `core` only. At the moment, the CSS Module `composes` output is recreated via the `form-common` import at the top of the [PIN SCSS](https://github.com/compassion-global-experience/compassion-design-system/blob/cf9549d0da582cab9fe0e6a3d2d386db0561e569/packages/core/src/components/Form/pin.scss).